styles.css

```diff
@@ -48,6 +48,53 @@ body {
   display: none;
 }
 
+/* hamburger menu */
+.hamburger-content {
+  display: block;
+  height: 87vh;
```

Comment:
- [ ] Good job following the project requirements but there is a small issue which can be fixed in no time. 
kindly consider giving this element `100vh` of height to make our design similar to the figma design as it is a requirement and also among the best practices to follow the figma design for better user experience.
your design
![re](https://user-images.githubusercontent.com/96683843/223495060-606b4141-d212-4d1e-85c2-cdc47cbb947a.jpg)

figma design
![qw](https://user-images.githubusercontent.com/96683843/223495044-9e9da4e4-cd70-4b6a-aada-cd1a04cf905e.jpg)